roles/database/tasks/initialize-fpsd-db.yml

```diff
@@ -3,27 +3,28 @@
   - name: Record present roles.
     command: psql -c "SELECT rolname FROM pg_roles;"
     register: postgres_roles
+    always_run: true
     changed_when: false
 
   - name: Create PostgreSQL user with user-defined password.
     postgresql_user:
-      name: fp_user
-      password: "{{ password }}"
-    when: "'{{ psql_env.PGHOST }}' not in postgres_roles.stdout and password"
+      name: "{{ fpsd_database_psql_env.PGUSER }}"
```

Comment:
The new logic allows customization of the postgres username, determined by the `fpf_database_psql_env` dict var. That seems consistent with the flow of the role, rather than wanting a hardcoded string.